### ISSUE TITLE
Consider storing passwords encrypted

### ISSUE THREAD

**Issue by firebus** [firebus/splunk-jira#18]
Per http://www.georgestarcher.com/2014/02/16/splunk-alert-scripts-automating-control/

**Comment by dhether** [comment]
+1 this enhancement also

**Comment by zjiaran** [comment]
How does putting them in a REST Endpoint a la - http://blogs.splunk.com/2011/03/15/storing-encrypted-credentials/ sound? If sufficient I'll add this functionality.

**Comment by firebus** [comment]
Yeah - that's the same blog posted referred to by the now 404 starcher link in the OP. +1.

**Comment by zjiaran** [comment]
Done deal. I'll get this and a setup.xml in soon.

**Comment by zjiaran** [comment]
As the setup.xml in default form writes to an admin only endpoint, I will be creating a custom endpoint that saves the passwords encrypted using the openssl included in Splunk.
